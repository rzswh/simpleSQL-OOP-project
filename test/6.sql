CREATE DATABASE OOP;
USE OOP;
CREATE TABLE qwe(asd DOUBLE, rty INT, zxc CHAR, PRIMARY KEY(asd));
INSERT INTO qwe(rty, zxc, asd) VALUES (1, "a", 2.1233242398479238098230948329084347897878);
INSERT INTO qwe(zxc, asd, rty) VALUES ("b", 1.1, 3);
INSERT INTO qwe(asd, rty, zxc) VALUES (4.2, 5, "c");
select * from qwe;
UPDATE qwe SET asd=5 WHERE zxc="a";
select * from qwe;
DROP DATABASE OOP;
