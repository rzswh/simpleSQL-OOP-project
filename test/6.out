asd	rty	zxc
1.1000	3	b
2.1233	1	a
4.2000	5	c
asd	rty	zxc
1.1000	3	b
4.2000	5	c
5.0000	1	a
